src/tools/generate_root_manifest.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Generates an Arcs root manifest from command line arguments. An import is
+# added for all command line args passed in that end with .arcs (files not
+# ending in .arcs are ignored). The root manifest is output to stdout.
+#
+# This script is designed to be used by the arcs_manifest_bundle BUILD rule.
+#
+# Usage: generate_root_manifest.sh \
+#     path/to/generating:rule \
+#     path/to/one.arcs \
+#     path/to/two.arcs \
+#     path/to/irrelevent.file
+
+echo "// Root manifest generated by $1"
+
```

Comment:
I kind of feel like this would be better done as a ctx.expand_template in a rule with python code doing the generation/concat of the import statements, instead of invoking a shell. I mean, it's not a big deal, by forking shells raises the risk of windows issues, and probably slows down the build a tiny bit.

Reply:
I reckon it's likely we'll move away from this generated root manifest idea soon anyway. But yeah, if not, we can move to a more complex rule fairly easily.